src/lib/account/evm.ts

```diff
@@ -58,7 +58,15 @@ export const deriveEvmAccountIdsAndMetadata: DeriveAccountIdsAndMetadata = async
     const bip44Params = adapter.getBIP44Params({ accountNumber })
 
     // use address if we have it, there is no need to re-derive an address for every chainId since they all use the same derivation path
-    address = address || (await adapter.getAddress({ accountNumber, wallet }))
+    address =
+      address ||
+      (await adapter.getAddress({ accountNumber, wallet }).catch(e => {
+        console.error(
+          `Failed to get address for account ${accountNumber} on chainId ${chainId}:`,
+          e,
+        )
+        return ''
+      }))
```

Comment:
Not really required - native and the like simply do "device" derivation, while MM `ethGetAddress` does an *internal* JSON-RPC request, should never fail, but added for the stake of completeness.